jenkins/bootstrap.py

```diff
@@ -967,6 +975,23 @@ def parse_args(arguments=None):
         '--clean',
         action='store_true',
         help='Clean the git repo before running tests.')
+    # Use "_no_-magic-env" because the default (current functionality)
+    # is to have magic environment, so the default value for this flag
+    # (false) needs to match this behavior.
+    parser.add_argument(
+        '--raw-env',
+        action='store_true',
+        help='Disable Jenkins magic environment')
```

Comment:
This isn't about jenkins, as we need it for non-jenkins too... maybe say remove env specific to the kubernetes project?